### ISSUE TITLE
Add a Yarn command to transpile JavaScript for IE11 testing

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#401]
# Feature Request

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## Is this feature request relating to an existing component or utility? Please describe.

- Component/utility name: The build step ?

## Do you have a suggestion for a new component or utility?
IE11 throws an error when I try to load our Storybook developer previews. I was testing the web component version of our accordions today ( https://github.com/department-of-veterans-affairs/vets-website/pull/15941 ) and IE wouldn't load. Checking the console, `vendors~main.BUILD_NUM.bundle.js` was throwing a console error because IE didn't understand the arrow function syntax.

If there was a Yarn command to transpile to ES5, we could test more easily in local environments.

## Acceptance Criteria
- [ ] Create and/or modify unit tests for this component to include an aXe scan

**Comment by CBonade** [comment]
Some updates as to the status of this:

I've gotten IE errors sorted on the `master` branch and there's a PR open for that here: https://github.com/department-of-veterans-affairs/component-library/pull/183 .  Essentially, the `CustomEvent` function in Javascript was causing an error. While we have a polyfill for this in `vets-website`, that doesn't cover our local testing environment since this is a separate repository. Copying this over to our `analytics.js` file to cover for the events, has prevented the React components from throwing errors when interactive elements that tracked analytics were clicked on.

Unfortunately I was unable to make the same level of success happen in the `web-components` branch.  I attempted to copy the config we have on the master branch as that seemed to work great, however on the master branch storybook is handling the build, while on the `web-components` branch it has to account for Stencil's build process. Stencil has a built-in `buildEs5` config option that's currently set to `prod` only. I tried setting that to `true` instead to have it apply to the dev environment, however it seems the error still persists. Additionally, just since the error seemed to be while opening storybook, I tried two different IE11 compatible storybook `yarn` add-ons, however neither of those worked either. The error we are getting here simply states `NotSupportedError` without much more information on-screen or in the console. I've been unable to locate other people struggling with the same error in a Storybook/Stencil setup, and even searching for that error alone doesn't produce much for me either. I am unsure of which way to go, so leaving this here hoping someone may have guidance on the direction to go forward.

**Comment by bkjohnson** [comment]
Closing since we no longer support IE11.
